modules/offers/client/views/offer-host-edit.client.view.html

```diff
@@ -124,6 +124,30 @@ <h4 id="maxGuests">How many guests can you accommodate?</h4>
                     </span>
                   </div>
                 </fieldset>
+
+                <br /><br />
+
+                <fieldset
+                  class="offer-showOnlyInMyCircles"
```

Comment:
No need `class` here since there is no CSS applying to this CSS selector either.